org-roam.el

```diff
@@ -162,11 +162,15 @@ If called interactively, then PARENTS is non-nil."
     (org-roam--build-cache-async)
     (user-error "Your Org-Roam cache isn't built yet! Please wait")))
 
-(defun org-roam--org-roam-file-p ()
-  "Return t if file is part of org-roam system, false otherwise."
-  (and (buffer-file-name (current-buffer))
-       (f-descendant-of-p (file-truename (buffer-file-name (current-buffer)))
-                          (file-truename org-roam-directory))))
+(defun org-roam--org-roam-file-p (file)
```

Comment:
My personal preference would be to make `file` an optional argument. If it is not present then use `buffer-file-name` instead. That will make it easier in debugging and make the function call cleaner.

Reply:
+1